jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/DefaultHandler.java

```diff
@@ -62,8 +62,8 @@ public class DefaultHandler extends Handler.Abstract
     private final long _faviconModifiedMs = (System.currentTimeMillis() / 1000) * 1000L;
     private final HttpField _faviconModified = new PreEncodedHttpField(HttpHeader.LAST_MODIFIED, DateGenerator.formatDate(_faviconModifiedMs));
     private ByteBuffer _favicon;
-    private boolean _serveFavIcon = true;
-    private boolean _showContexts = true;
+    private boolean _serveFavIcon;
+    private boolean _showContexts;
```

Comment:
The defaults in XML is `true`, so should be `true` here too (like it was before), to have the same behavior between standalone and embedded.

Reply:
It is either passed into the constructor or set to true by the default constructor, so it already defaults to true.